storage/object/how-to/get-started-edge-services.mdx

```diff
@@ -0,0 +1,293 @@
+    - Manually import a certificate into Scaleway Secret Manager:
+      - Enter a name for your certificate (alphanumeric characters only)
+      - Optionally, add tags by typing each tag and then pressing enter
+      - Copy and paste the full PEM-formatted certificate chain into the box.
+      Your certificate will be automatically stored in Secret Manager and billed accordingly. For Edge Services Private Beta testers, a credit of TODO will be applied to your Scaleway account for Secret Manager storage.
+
+    <Message type="tip">
+    For help with creating an an SSL/TLS certificate, and/or dealing with any errors you encounter when importing it into Secret Manager, see our [dedicated documentation](/storage/object/reference-content/ssl-tls-certificate/)
+    </Message>
+
+9. Click **Customize domain** to finish.
+
+Your customized domain is set up, and you are returned to the Edge Services dashboard. The customized domain displays in the Endpoint panel. When you access your bucket through this domain, it's content will be served via Edge Services.
+
+### How to edit your customized domain or its certificate
+
+After customizing your domain, you can edit it (or its certficate) at any time as follows:
+
+1. Click **Object Storage** in the **Storage** section of the Scaleway console side menu. The list of your buckets displays.
+
+2. Click the name of the bucket you want to configure a custom domain for.
+
+3. Click the **Edge Services** tab.
+
+    TODO SCREENSHOT
+
+4. In the **Endpoint** panel, click **Edit**. The following screen displays:
+
+    TODO SCREENSHOT
+
+5. Edit the subdomain as desired - do not forget to also set up a new CNAME record if necessary. 
+
+6. Select a different certificate from Secret Manager, or manually import a new certificate for your custom domain.
+
+7. Click **Edit domain** to finish.
+
+### How to reset your customized domain to the original Edge Services endpoint
+
+Even though the original Edge Services endpoint (e.g. `https://bucket-name.edge.scw.cloud`) will continue to work after you add a customized domain, you can choose to remove your customized domain completely and go back to the original Edge Services endpoint only. This is done via the reset function:
+
+1. Click **Object Storage** in the **Storage** section of the Scaleway console side menu. The list of your buckets displays.
+
+2. Click the name of the bucket you want to configure a custom domain for.
+
+3. Click the **Edge Services** tab.
+
+    TODO SCREENSHOT
+
+4. In the **Endpoint** panel, click **Reset**. 
+
+    A screen displays warning you that this will reset the bucket's domain back to the default Edge Services endpoint. Edge Services will consider your customized subdomain as unknown. You should also remember to delete your CNAME record from your domain provider, and your SSL/TLS certificate.
```

Comment:
@RoRoJ delete certificate from secret manager if not required anymore to avoid being billed